ui/page/send/page.go

```diff
@@ -163,34 +166,54 @@ func (pg *Page) OnNavigatedTo() {
 	currencyExchangeValue := pg.WL.MultiWallet.ReadStringConfigValueForKey(dcrlibwallet.CurrencyConversionConfigKey)
 	if currencyExchangeValue == values.USDExchangeValue {
 		pg.usdExchangeSet = true
-		pg.fetchExchangeValue()
+		go pg.fetchExchangeRate()
 	} else {
 		pg.usdExchangeSet = false
 	}
 	pg.Load.SubscribeKeyEvent(pg.keyEvent, pg.ID())
 }
 
-func (pg *Page) fetchExchangeValue() {
-	pg.exchangeError = ""
-	go func() {
-		var dcrUsdtBittrex load.DCRUSDTBittrex
+func (pg *Page) fetchExchangeRate() {
+	var dcrUsdtBittrex load.DCRUSDTBittrex
+	errText := "failed to fetch dcrUsdtBittrex value"
+
+	maxAttempts := 7
+	delayBtwAttempts := 2 * time.Second
+	attempts, err := components.RetryFunc(maxAttempts, delayBtwAttempts, func() error {
 		err := load.GetUSDExchangeValue(&dcrUsdtBittrex)
 		if err != nil {
-			pg.exchangeError = err.Error()
-			return
+			if strings.Contains(err.Error(), "no such host") {
```

Comment:
This string comparison is not reliable, best to do without it and always assume internet connection issues. Secondly, no need to display an error message each time `load.GetUSDExchangeValue` returns an error. Just return the error and let the retrials continue to  happen in the background. Only show an error message when the rate cannot be fetched after the maxAttempts.